### ISSUE TITLE
Build failure: Error: Unable to find 'guile_scm.swg'

### ISSUE THREAD

**Issue by ErikSobel-TRI** [RobotLocomotion/drake#2190]
Tried to do a fresh clone/build on Mac OS X.  Build failed.
Making install in swig
:3: Error: Unable to find 'guile_scm.swg'
./nlopt.i:15: Error: Unable to find 'std_vector.i'
make[5]: *** [nlopt-guile.cpp] Error 1
make[4]: *** [install-recursive] Error 1
make[3]: *** [nlopt-prefix/src/nlopt-stamp/nlopt-build] Error 2
make[2]: *** [CMakeFiles/nlopt.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [all] Error 2

**Comment by liangfok** [comment]
I haven't encountered this problem on my working OS X machine but here's what I see on my machine in case it helps to compare yours against mine:

``
$ cd [drake build artifacts directory]
$ find . -name "guile_scm.swg"
./build/share/swig/3.0.9/guile/guile_scm.swg
./externals/swig_matlab/swig-matlab/Lib/guile/guile_scm.swg
```

**Comment by liangfok** [comment]
Have you tried to turn on the `DISABLE_MATLAB` option when you run `make options`?

**Comment by ErikSobel-TRI** [comment]
Thanks for the suggestions:
1) Yes, I have the guile_scm.swg files in exactly the same location as you
2) Yes, I have turned ON the "DISABLE_MATLAB" flag

make options also says 
matlab :  not-found  
because there is no matlab installed on my system

But the error remains.

**Comment by liangfok** [comment]
Not sure if this is feasible but could you try completely removing Mac Ports from your machine?

**Comment by david-german-tri** [comment]
@ErikSobel-TRI is this still an open issue?

**Comment by ErikSobel-TRI** [comment]
Nope. I solve that.

But I'm not 100% sure I remember what the solution was.  One thing we fixed
was the PYTHON_VERSION MAJOR MINOR not being specified correctly possibly
in drake/bindings/swig/CMakeLists.txt (it just specified python2 and not
python2.7)

Then there was a separate issue where cmake was picking up the wrong VTK
because I have built VTK-7.0 from source on my Mac as well.  But I think
that was a separate issue.

-Erik
